api/StorageManager.json

```diff
@@ -5,34 +5,56 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/StorageManager",
         "support": {
           "webview_android": {
-            "version_added": null
+            "version_added": "48"
           },
           "chrome": {
-            "version_added": null
+            "version_added": "48"
           },
           "chrome_android": {
-            "version_added": null
+            "version_added": "48"
           },
           "edge": {
             "version_added": null
           },
           "edge_mobile": {
             "version_added": null
           },
-          "firefox": {
-            "version_added": null
-          },
+          "firefox": [
+            {
+              "version_added": "57"
+            },
+            {
+              "version_added": "51",
+              "version_removed": "57",
+              "notes": "See <a href='https://bugzil.la/1304966'>Bug 1304966</a> and <a href='https://bugzil.la/1399038'>Bug 1399038</a>.",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "dom.storageManager.enabled",
+                  "value_to_set": "true"
+                }
+              ]
+            }
+          ],
           "firefox_android": {
-            "version_added": null
+            "version_added": "51",
+            "notes": "See <a href='https://bugzil.la/1304966'>Bug 1304966</a> and <a href='https://bugzil.la/1399038'>Bug 1399038</a>.",
+            "flags": [
+              {
+                "type": "preference",
+                "name": "dom.storageManager.enabled",
+                "value_to_set": "true"
```

Comment:
MDN did not mention yet that the feature is still be behind a flag on Android:
https://github.com/mozilla/gecko-dev/blob/04b9cbbc2be2137a37e158a5ebaf9c7bef2364f9/modules/libpref/init/all.js#L5723-L5727